internal/authorization/resource_manager.go

```diff
@@ -0,0 +1,79 @@
+package authorization
+
+import (
+	"context"
+)
+
+func (a *Authorizer) createTuple(ctx context.Context, resourceID string) error {
+	user, ok := ctx.Value(USER_CTX).(*User)
+	if !ok {
+		// Should we panic?
+		return nil
+	}
+	go func() {
+		err := a.client.WriteTuple(context.Background(), "user:"+user.ID, CAN_VIEW, resourceID)
+		if err != nil {
+			a.logger.Errorf("Failed to create authorization tuple: %s", err)
+		}
+	}()
```

Comment:
we can't do this, have a look at how we use the [pool](https://github.com/canonical/identity-platform-admin-ui/blob/main/internal/pool/pool.go) in [here](https://github.com/canonical/identity-platform-admin-ui/blob/main/pkg/roles/service.go#L286-L289)

reason is this is directly affected by the number of requests on a Create operation, which would break the limit of file descriptors if put under stress

Reply:
I switched it to use the pool. It feels like the pool was not supposed to work like this (async jobs that we don't wish to wait for), but I guess it works as a temporary workaround.